composer.json

```diff
@@ -6,7 +6,7 @@
   "license": "MIT",
   "require": {
     "php": ">=7.1",
-    "php-amqplib/php-amqplib": "2.9.0",
+    "php-amqplib/php-amqplib": "^2.6",
```

Comment:
^ is pulling 2.7 and 2.8 and 2.9?

Reply:
Yes, we overwrote the problem on core level, now it's compatible with whatever "BC" stuff they throw at us when it comes to heartbeat